source/common/router/header_formatter.cc

```diff
@@ -112,6 +112,33 @@ parseUpstreamMetadataField(absl::string_view params_str) {
   };
 }
 
+std::function<std::string(const Envoy::RequestInfo::RequestInfo&)>
+parseStartTimeField(absl::string_view params_str, DateFormatterSharedPtr& date_formatter) {
```

Comment:
Can we share the same parse logic with the access log formatter for START_TIME here? I don't think there's any reason these need to be different.

Reply:
Will have a closer look.